app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html

```diff
@@ -79,6 +80,7 @@ <h2 class="heading-medium">@messages("results.two.years.old.guidance.title")</h2
 
         @changesToCircumstancesContent
```

Comment:
We need logic around here to only show this when we have at least 1 green. While the assumption that we would never hit /results without at least 1 GREEN is correct, a user could easily scenario play and get browse directly to /result with an all negative scenario, at which point is would be wrong.